fcrepo-http-api/src/main/resources/views/common-node-actions.vsl

```diff
@@ -50,13 +50,23 @@
     <input type="text" id="new_id" placeholder="(auto-generated identifier)" name="slug"  class="form-control"/>
         </div>
 
+    <div class="form-group" id="turtle_payload_container">
+        <label for="rdf"  class="control-label">
+	        RDF (turtle)
+        </label>
+        <textarea rows="10" id="turtle_payload" name="turtle" class="form-control" 
+                  placeholder="@prefix dcmitype: <http://purl.org/dc/dcmitype/> 
+<> a dcmitype:Collection ."></textarea>
```

Comment:
I've been looking into this, too, and all the mechanisms I've seen for including multi-line text in placeholder attributes seem pretty flaky. I'd either remove the attribute or put in something like this: `<> a <http://purl.org/dc/dcmitype/Collection>` which doesn't require newlines.